selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/DefaultAndroidEmulator.java

```diff
@@ -143,14 +145,26 @@ public String getAvdName() {
 
   public static List<AndroidEmulator> listAvailableAvds() throws AndroidDeviceException {
     List<AndroidEmulator> avds = Lists.newArrayList();
+    String avdParam = "avds";
+    File clu = null;
+
+    if(AndroidSdk.getAndroidVersionNumber() >= SDK_VERSION_FOR_AVD_MANAGER ){
+      clu =  AndroidSdk.avdManager();
+      avdParam = "avd";
+    }
+    else {
+      clu = AndroidSdk.android();
+    }
+
+    CommandLine getAVDCommand = new CommandLine(clu);
+    getAVDCommand.addArgument("list", false);
+    getAVDCommand.addArgument(avdParam, false);
 
-    CommandLine cmd = new CommandLine(AndroidSdk.android());
-    cmd.addArgument("list", false);
-    cmd.addArgument("avds", false);
 
+    log.info("Executing command for avd: "+getAVDCommand.toString());
```

Comment:
No need to log here, `ShellCommand` already does this for you :)

Reply:
Removed logging !